### PR TITLE
Unifying docs on Napari DeepLabCut

### PR DESCRIPTION
**Pull request by Timokleia**
I updated the docs so the information and structure is unified.
I didnt touch the installation process at all (a bit different display between the two).
I deleted some old commits of mine - no relationship with the current work but they appear in the list.

### CODE REVIEW

File: docs/napariGUI.md [hunk 1]
```diff
@@ -108,16 +112,57 @@ Suggested workflows, depending on the image folder contents:
 
     Saving works as described in *1*.
 
-3. **Refining labels** – the image folder contains a `machinelabels-iter<#>.h5` file.
+    ***Note that if a new body part has been added to the `config.yaml` file after having started to label, loading the config in the GUI is necessary to update the dropdown menus and other metadata.***
+
+    ***As `viridis` is `napari-deeplabcut` default colormap, loading the config in the GUI is also needed to update the color scheme.***
```
Comment (MMathisLab) [inline docs/napariGUI.md]: I think this is now out of date with the new color picker, right @jeylau ?

**Review by MMathisLab — CHANGES_REQUESTED** [review]
- maybe check this one point, 
 - @Timokleia also lets link the new hotkey image in the docs?

**Review by MMathisLab — APPROVED** [review]

**Comment by Timokleia** [comment]
@MMathisLab I am working on it this week. I will update - pls dont accept yet

**Comment by MMathisLab** [comment]
I already edited it and merged, thanks @Timokleia

**Pull request merged**